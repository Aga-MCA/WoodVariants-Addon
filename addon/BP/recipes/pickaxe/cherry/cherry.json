{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "cherry_cherry_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:cherry_stick"
      },
      "P": {
        "item": "aga_wood_variants:cherry_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:cherry_cherry_pickaxe"
    }
  }
}