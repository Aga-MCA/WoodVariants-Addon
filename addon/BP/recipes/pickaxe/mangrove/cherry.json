{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "mangrove_cherry_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:cherry_stick"
      },
      "P": {
        "item": "aga_wood_variants:mangrove_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:mangrove_cherry_pickaxe"
    }
  }
}