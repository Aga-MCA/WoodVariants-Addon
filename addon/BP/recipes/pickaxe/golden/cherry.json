{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_cherry_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:cherry_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_cherry_pickaxe"
    }
  }
}